### PR TITLE
Hypershift 0.6 changes

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Changed the name of the branch to remove the period, so the preview builds OK.

Replaces https://github.com/timescale/docs/pull/2410

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: use-timescale/migration/about-hypershift.md [hunk 1]
```diff
@@ -67,19 +67,30 @@ URIs in the configuration file.
 To see all available configuration options for hypershift, use this command:
 
 ```bash
-docker run -ti timescale/hypershift:0.5 --help
+docker run -ti timescale/hypershift:0.6 --help
 ```
 
+### The hypershift clone tool
+
+You can use the hypershift `clone` command to preserve a copy of your database
```
Comment (Harkishen-Singh) [inline use-timescale/migration/about-hypershift.md:75]:
```suggestion
You can use the hypershift `clone` command to make a copy of your database
```
Does `make` works? I think it simplifies the sentence.

File: use-timescale/migration/about-hypershift.md [hunk 2]
```diff
@@ -67,19 +67,30 @@ URIs in the configuration file.
...
+in the source location. With this command, instead of moving the contents off
+the source and into the target, hypershift makes a full copy of the contents of
+your source database, and moves it into the source database. You can add
+parameters to the `clone` command to filter which contents to copy.
+
```
Comment (Harkishen-Singh) [inline use-timescale/migration/about-hypershift.md:80]: Can we simplify to this?

```suggestion
in the source location. Hypershift makes a full copy of the contents of
your source database, and moves it into the target database. You can use
parameters in the `clone` command to filter which contents to copy.
```

File: use-timescale/migration/about-hypershift.md [hunk 3]
```diff
@@ -67,19 +67,30 @@ URIs in the configuration file.
...
 |Short command|Long command|Example use|Description|
 |-|-|-|-|
-|-c|--config-file|-C <CONFIG_FILE>|Path for hypershift configuration file|
 |-s|--source|-s <SOURCE_URI>|URI to the source database|
 |-t|--target|-t <TARGET_URI>|URI to the target database|
 ||--source-password|--source-password|Password for the source database|
 ||--target-password|--target-password|Password for the target database|
-||--ignore-missing-time-index||Ignore that hypertables may be missing a time index|
+||--parallel|--parallel <NUMBER\>|Number of workers that can simultaneously fetch data from the target database|
```
Comment (Harkishen-Singh) [inline use-timescale/migration/about-hypershift.md:87]: Is the `\` in `<NUMBER\>` intentional? There is also another one at 92, `<PATH\>`

**Review by harkishen — CHANGES_REQUESTED** [review]

File: use-timescale/migration/about-hypershift.md (hunk 1 above)
Comment (Loquacity) [reply use-timescale/migration/about-hypershift.md:75]: "make" implies creating something new, which I'm not sure is actually what is happening ...

**Review by VineethReddy02 — APPROVED** [review]
LGTM!

File: use-timescale/migration/about-hypershift.md (hunk 1 above)
Comment (Loquacity) [reply use-timescale/migration/about-hypershift.md:75]: "create" again, means something new.

Comment (Harkishen-Singh) [reply use-timescale/migration/about-hypershift.md:75]: Oh, then lets go with preserve. The reason why I suggested create is Hypershift is responsible to create a copy of the database to the target.

**Review by harkishen — APPROVED** [review]

**Pull request merged**